bundles/framework/myplaces3/ButtonHandler.js

```diff
@@ -255,13 +247,14 @@ Oskari.clazz.define("Oskari.mapframework.bundle.myplaces3.ButtonHandler",
          * @param {Object} data
          */
         featureAdded: function (data) {
-            if (this.dialog && this.drawMode) {
-                var areaDialogContent = this.loc('tools.' + this.drawMode + '.next'),
-                    content = this.dialog.getJqueryContent();
-                if (content.find('div.infoText') !== areaDialogContent) {
-                    content.find('div.infoText').html(areaDialogContent);
-                    this.dialog.moveTo('#toolbar div.toolrow[tbgroup=default-myplaces]', 'top');
-                }
+            if (!this.dialog || !this.drawMode) {
+                return;
+            }
+            var areaDialogContent = this.loc('tools.' + this.drawMode + '.next');
+            var content = this.dialog.getJqueryContent();
+            if (content.find('div.infoText') !== areaDialogContent) {
```

Comment:
This if doesn't actually make sense since we are comparing text string to a jQuery object.